deploy/server/server.properties

```diff
@@ -15,7 +15,7 @@ online-mode=true
 gamemode=1
 force-gamemode=true
 
-# Nether and end are not supported
+# Nether and finish are not supported
```

Comment:
This should stay as end

Reply:
Yea, thanks for catching, this was because I changed #end() to #finish() so it matches MatchPhase#FINISHED